### DISCUSSION TITLE
Ubuntuサーバー上に直接構築した場合、Hack MDとの連携はどのように行うのでしょうか。

### DISCUSSION THREAD

**Discussion by Derorin719** [growilabs/growi#8086] [Help / Questions] [answered]
公式ドキュメントではDocker composeについて言及されていました。直接構築した場合は可能ですか？
もしくは、Docker compose版に今のうちに乗り換えておいたほうが良いですか？

**Comment by yuki-takei** [comment, marked as answer]
自前で構築も可能です。こちらに沿って設定してください。

https://docs.growi.org/ja/admin-guide/admin-cookbook/integrate-with-hackmd.html#%E6%A6%82%E8%A6%81

とはいえ、growi-docker-compose の方がおすすめです。
